e2e-tests/tests/test-evolution.ts

```diff
@@ -147,4 +133,104 @@ describeWithExistingNode("Frontier RPC (Mint and Evolve Assets)", (context) => {
             )
         );
     });
+
+    
+});
+
+describeWithExistingNode("@qa Frontier RPC (Public Minting)", (context) => {
+    let collectionContract: Contract
+
+    beforeEach(async function () {
+        collectionContract = await createCollection(context);
+    });
+
+    step("when is transferred owner should change and emit an event", async function () {
+        const newOwner = "0xf24FF3a9CF04c71Dbc94D0b566f7A27B94566cac";
+
+        expect(await collectionContract.methods.owner().call()).to.be.eq(FAITH);
+        const tranferringResult = await collectionContract.methods.transferOwnership(newOwner).send({ from: FAITH, gas: GAS_LIMIT });
+        expect(tranferringResult.status).to.be.eq(true);
+        expect(await collectionContract.methods.owner().call()).to.be.eq(newOwner);
+
+        expect(Object.keys(tranferringResult.events).length).to.be.eq(1);
+
+        // data returned within the event
+        expect(tranferringResult.events.OwnershipTransferred.returnValues._previousOwner).to.be.eq(FAITH);
+        expect(tranferringResult.events.OwnershipTransferred.returnValues._newOwner).to.be.eq(newOwner);
+
+        // event topics
+        expect(tranferringResult.events.OwnershipTransferred.raw.topics.length).to.be.eq(3);
+        expect(tranferringResult.events.OwnershipTransferred.raw.topics[0]).to.be.eq(SELECTOR_LOG_OWNERSHIP_TRANSFERRED);
+        expect(tranferringResult.events.OwnershipTransferred.raw.topics[1]).to.be.eq(context.web3.utils.padLeft(FAITH.toLowerCase(), 64));
+        expect(tranferringResult.events.OwnershipTransferred.raw.topics[2]).to.be.eq(context.web3.utils.padLeft(newOwner.toLowerCase(), 64));
+        // event data
+        expect(tranferringResult.events.OwnershipTransferred.raw.data).to.be.eq('0x');
+
+        try { // TODO here check this error
+            await collectionContract.methods.transferOwnership(FAITH).send({ from: FAITH, gas: GAS_LIMIT });
+            expect.fail("Expected error was not thrown"); // Ensure an error is thrown
+        } catch (error) {
+            // console.log(error.message);
+        }
```

Comment:
Redundant TODO? The condition is checked with `expect.fail`.

Also, do you need the commented line `console.log(error.message)`? If  not needed we can remove it